src/main/java/seedu/address/logic/commands/StatisticsCommand.java

```diff
@@ -0,0 +1,98 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.Priority;
+
+/**
+ * Displays the overall statistics of the people in the addressbook.
+ */
+public class StatisticsCommand extends Command {
+
+    public static final String COMMAND_WORD = "statistics";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Displays the overall statistics regarding all the people in SocialBook.\n"
+            + "Example: " + COMMAND_WORD;
+    public static final String MESSAGE_DISPLAY_STATISTICS_SUCCESS = "Here are all the statistics:\n%s";
+    public static final String MESSAGE_DISPLAY_TOTAL_PEOPLE = "Total Number Of People: %s";
+    public static final String MESSAGE_DISPLAY_HIGH_PRIORITY = "Number Of HIGH Priority People: %s";
+    public static final String MESSAGE_DISPLAY_MEDIUM_PRIORITY = "Number Of MEDIUM Priority People: %s";
+    public static final String MESSAGE_DISPLAY_LOW_PRIORITY = "Number Of LOW Priority People: %s";
+    private String resultMessage;
```

Comment:
Instead of storing the result message here, perhaps can generate the statistics in the constructor and store it as a final value?